### ISSUE TITLE
Cant install extestions 

### ISSUE THREAD

**Issue by Ud4ba** [Eloston/ungoogled-chromium#970]
Ive trying everything . I tried unpacking and then installing .. Nothing works .. This a serious issue ....
A browser without add ons?
same as here #923 i already opened an issue but someone closed it .. this is a very serious bug .

**Comment by Eloston** [comment]
> I tried unpacking and then installing

Can you describe step-by-step how you tried to unpack and install extensions? A video works too.

**Comment by PF4Public** [comment]
Have you tried to right-click *.crx and 'open it with" → "Chromium" like @perfect7gentleman suggested in a [closed bug](https://github.com/Eloston/ungoogled-chromium/issues/964)?

I just did and it worked flawlessly.

**Comment by Arecsu** [comment]
Second this problem. Tried enabling Developer mode in chrome://extensions/, drag and drop an extension from Windows Explorer and the whole page freezes at Drop to Install.

Right clicking *.crx and open it with Ungoogled Chromium doesn't work as well. It gets stick on "Adding to Chromium..."

Here's a screenshot showing both problems. It happens with every extension. Windows 10 1909 x64 and Ungoogled Chromium 80.0.3987.132 x64

How can I log further this issue?

![image](https://user-images.githubusercontent.com/12679098/77362920-92faf700-6d30-11ea-969a-b1c22ad809cc.png)

**Comment by Ud4ba** [comment]
love you for this .

**Comment by Arecsu** [comment]
Ok, I unpacked the extension — uncompress the crx file into a folder — then I loaded as unpacked and it worked. Anyways, I installed the extension Chromium Web Store using this method but I'm still unable to install extensions from the Chrome Web Store or anywhere as long as it download/uses a *crx file. I have to unpack every single extension to get it working.

Would be great if this problem get fixed!
